utils/config/examples/daos_server_sockets.yml

```diff
@@ -86,3 +86,25 @@ servers:
   # bdev_list: [/tmp/daos-bdev]	# generate nvme.conf as follows:
               # [AIO]
               #   AIO /tmp/aiofile AIO1 4096
+-
+  targets: 8                # count of storage targets per each server
+  first_core: 25            # offset of the first core for service xstreams
+  nr_xs_helpers: 0          # count of offload/helper xstreams per target
+  fabric_iface: eth0        # map to OFI_INTERFACE=eth0
```

Comment:
eth1?

Reply:
that's what I have it configured as, should it be ibX?

Reply:
What I meant is that the first server configuration already has eth0, so I think this should either be eth1 or else the port needs to be different.

Reply:
yes will change port, probably good having an example that uses the same NIC, even if it is not the most performant configuration

Reply:
wait but won't the fabric interface being the same fail your new config validation?

Reply:
reverted to having different NICs in order not to encourage configurations that might affect performance